### PR TITLE
Order Model, Repository, Interface, and Controller

### PR DESCRIPTION
**Pull request by sarahemaginnis**
Created two routes, GetAllOrders and GetOrderById. Will need to update later on to join to PaymentMethod tables (joining with PaymentMethodId) and User tables (joining with UserId)

### CODE REVIEW

File: LoreStoreAPI/Controllers/OrderController.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        [HttpPut("{id}")]
+        public void Put(int id, [FromBody] string value)
+        {
+        }
+
+        // DELETE api/<OrderController>/5
+        [HttpDelete("{id}")]
+        public void Delete(int id)
+        {
+        }
```
Comment (GuessImNick) [inline LoreStoreAPI/Controllers/OrderController.cs:56]: Are we going to be finishing routes in a different PR?

Comment (sarahemaginnis) [reply LoreStoreAPI/Controllers/OrderController.cs:143]: Finished the routes and it's ready for review and merging!

**Review by GuessImNick — APPROVED** [review]
